README.markdown

```diff
@@ -99,7 +99,7 @@ To verify that nvm has been installed, do:
 command -v nvm
 ```
 
-which should output 'nvm' if the installation was successful. Please note that `which nvm` will not work, since `nvm` is a sourced shell function, not an executable binary.
+which should output 'nvm' if the installation was successful. Please note that `which nvm` will not work, since `nvm` is a sourced shell function, not an executable binary. Also note, you may need to close your existing terminal window and run the command in a new window.
```

Comment:
the install script itself already outputs this information - is it necessary here?

Also, I don't like the vagueness of "may" - either they need to, or they don't, and I'm pretty sure they always do if they used the install script.